src/main/java/model/schedules/BackOffice.java

```diff
@@ -0,0 +1,12 @@
+package model.schedules;
+
+import java.util.ArrayList;
+
+public class BackOffice extends Roster{

```

Comment:
Suggest rename of class to BackOfficeRoster so everyone knows that the Roster is meant?